keyboards/ymd75/rev1/config.h

```diff
@@ -27,3 +27,4 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define BACKLIGHT_PIN D4
 #define DIODE_DIRECTION COL2ROW
 #define DEVICE_VER      0x0100
+#define DYNAMIC_KEYMAP_LAYER_COUNT 3
```

Comment:
```suggestion
```

Reply:
@fauxpark There is a an compiler error for rev1 board without decreasing DYNAMIC_KEYMAP_LAYER_COUNT.

 error: #error Dynamic keymaps are configured to use more EEPROM than is available.


Does is needs to be fixed in via keymap folder config.h, or i should not worry about it and only add keymap.c and rules.mk?

Reply:
If it's complaining about that then you need to remove the extra layer in the keymap.c instead.

Reply:
I tried to delete 2  layers in keymap.c, but error persists. If i add config.h in via folder with 

#if DEVICE_VER == 0x0100
	#define DYNAMIC_KEYMAP_LAYER_COUNT 3
#endif

then everything is fine, but drashna said i need to move this code  to rev1 folder config.h instead.

What should i do?

Reply:
Sorry, I meant "as well", not "instead".

You *could* add VIA keymaps for all three revisions under their respective folders, and give only the rev1 keymap three layers + this define, but that seems a little messy.

Reply:
Honestly i don't know how to do it better and if it makes any sense to make VIA keymap available for all 3 revisions of board, because AFAIK VIA only looks for "vendorId"  and "productId". If wiring between revisions is different, then for now the only way is to separate them by "productId". 

Added define to VIA folder config.h in the end.